src/ripple/rippleTypes.ts

```diff
@@ -48,3 +49,35 @@ export type RipplePrivateKeys = ReturnType<typeof asRipplePrivateKeys>
 export const asRipplePrivateKeys = asObject({
   rippleKey: asString
 })
+
+export interface RippleNetworkLocation {
+  currency: string
+  issuer: string
+}
```

Comment:
You already have `asXrpNetworkLocation` a few lines above. You can do `type RippleNetworkLocation = ReturnType<typeof asXrpNetworkLocation>`

Reply:
Ah good catch. how'd i miss that.